src/Altinn.Apps/AppTemplates/AspNet/App/Startup.cs

```diff
@@ -187,6 +198,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseDeveloperExceptionPage();
             }
 
+            string applicationId = GetApplicationId();
```

Comment:
It is possible for applicationId to be an empty string. What then?

Reply:
If applicationId is empty, the documentation will be on the root of the server. 
Ex: http://altinn3local.no/swagger/index.html instead of http://altinn3local.no/ttd/apps-test-appref/swagger/index.html. 
So there will be a problem if we have multiple applications with empty applicationId. 
Maybe we have to genereate a random applicationId and log the problem ?

Reply:
There is probably no routing for `swagger` in the load-balancer anyway. Maybe it will result in 404? You can test that already in any environment. 

One option is to simply skip all the setup.

Reply:
Skip the setup seems to be the cleanest way.